test/17_main/test.js

```diff
@@ -53,5 +53,42 @@ define(['BlinkForms', 'BIC'], function (Forms) {
         _.each(Forms._models, defaultsIsAFunctionTest);
       });
     });
+
+    suite('isValid called', function () {
+      test('all possible options', function () {
+        var model = Forms._models.Element;
+        var element = new model();
```

Comment:
Probably need to change `model` to `Model` on these 2 lines to pass the ESLint rules.

Because there's otherwise no way to tell the difference between a JavaScript function and one that can be used as a constructor (with the `new`) keyword, we have a convention to give constructor functions a capitalised first letter.